scanomatic/io/scanstore.py

```diff
@@ -22,7 +22,7 @@ def add_scan(self, project_fullname, scan):
             scan.timedelta.total_seconds(),
         )
         path = os.path.join(self._path, project_fullname, filename)
-        with open(path, 'w') as f:
+        with open(path, 'wb') as f:
```

Comment:
Interesting! I wasn't aware this was different in Python 3, but I guess it makes sense with the new string/bytes division.

Reply:
the difference is that what we attempt to write is now bytes so we need to write it as bytes too.

Reply:
Yes, I hadn't registered that this was the case in 3, but it makes sense.

Reply:
Yes, I hadn't registered that this was the case in 3, but it makes sense.